### PR TITLE
doc(news): add missing news entries

### PR DESCRIPTION
**Pull request by alissonlauffer**
If there's anything missing, just let me know. It's been a while since we filed any news entries, and they'll be needed for the next release.

### CODE REVIEW

File: news/+misc.misc.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Misc fixes, improvements and micro optimizations.
```
Comment (HitaloM) [inline news/+misc.misc.rst:1]: Don't you think it would be better to list the fixes and optimizations made?

**Review by HitaloM — COMMENTED** [review]
Everything seems good to me 👏

File: news/+misc.misc.rst (hunk 1 above)
Comment (alissonlauffer) [reply news/+misc.misc.rst:1]: Well, this news fragment is for the things that the end-users probably won't care about. If you think that any of these changes deserve their own news  fragment, we may add it.

Comment (HitaloM) [reply news/+misc.misc.rst]: > Well, this news fragment is for the things that the end-users probably won't care about. If you think that any of these changes deserve their own news fragment, we may add it.

Good point lol

**Review by HitaloM — APPROVED** [review]

**Pull request merged**